.github/workflows/pull-request.yaml

```diff
@@ -21,7 +21,10 @@ jobs:
 
       - name: Pull in all relevant branches
         run: git fetch origin release-v2.6
```

Comment:
Should we update this to `release-v2.8`?

Reply:
We should. I'll open a different PR as that change is not related to this one